config/prow/cluster/monitor-prow-controller_pod.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: monitor-prow-controller
+  namespace: prow
+spec:
+  serviceAccountName: monitor-prow-controller
+  containers:
+  - name: monitor-prow-controller
+    image: quay.io/powercloud/all-in-one:0.2
+    command:
+      - /bin/bash
+    args:
+      - -c
+      - |
+        KCTL="kubectl -n prow"
+        while true;do
+            $KCTL logs deployment/prow-controller-manager | grep "Failed to get API Group-Resources"
+            if [[ $? -eq 0 ]]; then
+                $KCTL scale deployment prow-controller-manager --replicas=0
+                $KCTL scale deployment prow-controller-manager --replicas=1
+            fi
+            sleep 10
```

Comment:
I feel it's too frequent running in the cloud, can we bump it to 2mins or 5mins?